scripts/postbuild.sh

```diff
@@ -5,5 +5,5 @@ set -e
 # Generate sitemap
 pnpm exec next-sitemap
 
-# Update blog search data
-curl -o public/feed.json  https://authzed.com/feed/json
```

Comment:
This is a meaningful change - with the new `pagefind` approach to search, there isn't a good way to include blog results and have them be sorted nicely against docs results. I decided to remove them for the time being.